app/characteristics_of_a_knight/knight.py

```diff
@@ -0,0 +1,25 @@
+class Knights:
+    @staticmethod
+    def characteristics(knights_configs: dict) -> dict:

```

Comment:
In case a class consists of only one static method, you could create it as a separate function and not create a class for it. Besides that, function names should describe actions in the form of a verb. The same comment with the static method relates to the `BattleKnights` class as well.